sdk/core/core-lro/src/poller.ts

```diff
@@ -379,13 +362,37 @@ export abstract class Poller<TState extends PollOperationState<TResult>, TResult
     return this.pollOncePromise;
   }
 
+  private processUpdatedState(): void {
+    if (this.operation.state.error) {
+      this.stopped = true;
+      this.reject!(this.operation.state.error);
```

Comment:
any reason to have this be `this.reject!` instead of `this.reject?`

Reply:
`this.reject` is initialized in the constructor (L255) so we know for sure it can't be `undefined`:
https://github.com/Azure/azure-sdk-for-js/blob/c59084407bf15d4f9c8d336eb3d73f36496a4766/sdk/core/core-lro/src/poller.ts#L247-L255